dashboard/api.py

```diff
@@ -1 +1,92 @@
 import requests
+
+def get_visit_events(project_name, after_time, limit, after_id=None)
+    url = "analytics.bitsofgood.org/api/events/visit-events"
```

Comment:
I would probably factor the base url out to be a global variable

so you would have something like 
```
base_url = "https://analytics.bitsofgood.org"

def get_paginated_visit_events():
    url = base_url + "/api/events/visit-events"
```